### ISSUE TITLE
Remote branch 2.0.0 not found in upstream origin

### ISSUE THREAD

**Issue by Phestek** [jaagr/lemonbuddy#75]
Hi,
```
[damian@archie][~]% git clone --branch 2.0.0 --recursive https://github.com/jaagr/lemonbuddy
Cloning into 'lemonbuddy'...
fatal: Remote branch 2.0.0 not found in upstream origin
```
I think, your README.md have some mistakes and typos.

**Comment by jaagr** [comment]
Oh, thanks for reporting it.. Replace `2.0.0` with `2.0.0-beta` and you should be good to go

**Comment by jaagr** [comment]
fixed bb8fc67c95b0e70f948d450497b53ca63200e0ec
